src/cryptoadvance/specter/util/common.py

```diff
@@ -29,6 +29,53 @@ def snake_case2camelcase(word):
     return "".join(x.capitalize() or "_" for x in word.split("_"))
 
 
+def replace_substring(text, start_position, replace_length, new_str):
+    return text[:start_position] + new_str + text[start_position + replace_length :]
+
+
+def satamount_formatted(value):
+    return "{:,.0f}".format(round(float(value) * 1e8))
+
+
+def btcamount_formatted(
+    value,
+    maximum_digits_to_strip=7,
+    minimum_digits_to_strip=6,
+    enable_digit_spaces_for_counting=True,
+):
+    value = round(float(value), 8)
+    formatted_amount = "{:,.8f}".format(value)
+
+    count_digits_that_can_be_stripped = 0
+    for i in reversed(range(len(formatted_amount))):
+        if formatted_amount[i] == "0":
+            count_digits_that_can_be_stripped += 1
+            continue
+        break
+
+    if count_digits_that_can_be_stripped >= minimum_digits_to_strip:
+        for i in reversed(range(len(formatted_amount))):
+            if (
+                formatted_amount[i] == "0"
+                and len(formatted_amount) - i <= maximum_digits_to_strip
+            ):
+                # replace with https://unicode-table.com/en/2007/
+                formatted_amount = replace_substring(formatted_amount, i, 1, " ")
```

Comment:
Why not use `\u2007` in
<img width="511" alt="grafik" src="https://user-images.githubusercontent.com/70536101/187511042-8b729d30-a097-48a5-abf3-cca629ad7282.png"> ?
Thus:` replace_substring(formatted_amount, i, 1, "\u2007")`
As you can see, GitHub's front end is already not even showing the figure space. Same for below with `\u2008`.

Reply:
Also: Did you have specific reasons to choose those unicode space types? I just found out that there are 20 different types.

Reply:
Yes, I chose the spaces such, that they are equivalent in width to a number https://unicode-table.com/en/2007/  https://en.wikipedia.org/wiki/Figure_space and equivalent to a Punctuation https://unicode-table.com/en/2008/. 

> Why not use `\u2007` in

I'll try that.

Reply:
Cool. As for Cypress, just ignore it for now, it's just flakiness.